custom_components/miele/strings.json

```diff
@@ -284,7 +284,7 @@
                     "latte_macchiato": "Latte macchiato",
                     "espresso_macchiato": "Espresso macchiato",
                     "cafe_au_lait": "Café au lait",
-                    "caffellatte": "Caffellatte",
+                    "caffe_latte": "Caffellatte",
```

Comment:
If that's the English "translation" (or rather: The original strings without any translation), I think it should be `Caffè latte` as per https://en.wikipedia.org/wiki/Latte 
<img width="754" alt="image" src="https://user-images.githubusercontent.com/6943905/231850654-1279508e-00bb-4ff5-a8cb-7d7f6fc856dd.png">